trade-finance-manager-ui/server/controllers/case/parties/index.js

```diff
@@ -145,6 +147,34 @@ const postPartyDetails = (partyType) => (
       return res.redirect('/not-found');
     }
 
+    /**
+     * checks if partyType is exporter and if partyUrn input exists or is a blank string (nothing inputted)
+     * validates input - should only be at least 3 numbers
+     * validation error rendered if error exists on edit partyUrn page
+     * if no error, then updates partyURN
+     */
+    if (partyType === 'exporter' && (req.body.partyUrn || req.body.partyUrn === '')) {
```

Comment:
You can use a helper function for string which ascertain whether string exists and is not null or undefined

Reply:
done